#! /usr/bin/env python

PACKAGE='pr2_interactive_manipulation'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()


gen.add("reactive_grasping", bool_t, 1, "Reactive Grasping", False)
gen.add("lift_steps", int_t, 1, "Lift After Grasp (cm)", 10, 0, 10)
gen.add("lift_vertically", bool_t, 1, "Lift Vertically", True)

exit(gen.generate(PACKAGE, "interactive_manipulation_marker_node", "Pickup"))
